[Strings] Renamed "world map" to  "world overview map" for better user clarity

hemanggs opened the pull request:
Fixes #9334 


Strings changed : 

1.  Before : `Download the world map`
     After: `Download the world overview`

2. Before : `Before you start using the app, please download the general world map to your device.`
    After: `Before you start using the app, please download the general world overview to your device.`


"world overview" seems to be more adequate avoiding confusion, feel free to share better alternatives 


- Updated corresponding translations using DeepL API as mentioned in [Translations.md](https://github.com/organicmaps/organicmaps/blob/master/docs/TRANSLATIONS.md) 
- The Updated translated string seems to cause error when running `tools/unix/generate_localizations.sh` , for the languages `fa` &  `zh-Hant` , so i reverted these strings to original version
- Regenerated localizations with `tools/unix/generate_localizations.sh`

Jean-BaptisteC commented on data/strings/strings.txt:
```diff
@@ -1571,50 +1571,49 @@
+    eu = Aplikazioa erabiltzen hasi aurretik, deskargatu munduko ikuspegi orokorra gailura.\n%@ Biltegiratze erabiliko du.
+    fa = قبل از شروع استفاده از برنامه ، لطفاً نمای کلی جهانی را در دستگاه خود بارگیری کنید.\nاز ٪@ ذخیره سازی استفاده می کند.
+    fi = Ennen kuin aloitat sovelluksen käytön, lataa laitteeseesi yleiskatsaus maailmasta.\nSe käyttää %@ tallennustilaa.
+    fr = Avant de commencer à utiliser l'application, veuillez télécharger la présentation générale du monde sur votre appareil.\nIl utilisera %@ de l'espace de stockage.
```
`fr = Avant de commencer, permettez-nous de télécharger la carte générale du monde dans votre appareil.\n%@ de données sont nécessaires.`

Old translation is better

Jean-BaptisteC commented on data/strings/strings.txt:
```diff
@@ -29965,41 +29964,49 @@
+    eu = Deskargatu munduko ikuspegi orokorra
+    fa = نمای کلی جهان را بارگیری کنید
+    fi = Lataa maailman yleiskatsaus
+    fr = Téléchargez l'aperçu du monde
```
`fr = Télécharger la carte générale du monde`

hemanggs commented:
Sorry about the messy commit history , initially i had some issues with DCO signs.

matheusgomesms commented on data/strings/strings.txt:
```diff
@@ -1571,49 +1571,48 @@
+    nb = Før du begynner å bruke appen, må du laste ned den generelle verdensoversikten til enheten din.\nDet vil bruke %@ av lagringsplassen.
+    nl = Download het algemene wereldoverzicht naar uw apparaat voordat u de app gaat gebruiken.\nHet zal %@ opslagruimte gebruiken.
+    pl = Przed rozpoczęciem korzystania z aplikacji proszę pobrać ogólny przegląd świata na swoje urządzenie.\nWykorzysta %@ pamięci masowej.
+    pt = Antes de começar a utilizar a aplicação, descarregue a visão geral do mundo para o seu dispositivo.\nUtilizará %@ do armazenamento.
+    pt-BR = Antes de você começar a usar o aplicativo, faça o download da visão geral do mundo para o seu dispositivo.\nEle usará %@ do armazenamento.
```
```suggestion
    pt = Antes de começar, é recomendável descarregar a visão geral mundial para o seu dispositivo.\nÉ necessário %@ de espaço disponível.
    pt-BR = Antes de começar, vamos baixar a visão geral mundial para o seu dispositivo.\n%@ É necessário %@ de armazenamento.
```

matheusgomesms replied:
PT and PT-BR review

hemanggs replied:
The PT - BR you suggested has  error when generating localizations , i will stick to the old one for now.

map-per replied:
There are two "%@" in the string, the first one needs to be removed:

`pt-BR = Antes de começar, vamos baixar a visão geral mundial para o seu dispositivo.\nÉ necessário %@ de armazenamento.`

hemanggs replied:
okay got it ,  ill make the change , i thought the same but assumed it would affect the meaning somehow .

pastk commented on data/strings/strings.txt:
```diff
@@ -1585,7 +1585,7 @@
     et = Enne rakenduse kasutamise alustamist laadige oma seadmesse alla üldine maailma ülevaade.\nSee kasutab %@ salvestusruumi.
     eu = Aplikazioa erabiltzen hasi aurretik, deskargatu munduko ikuspegi orokorra gailura.\n%@ Biltegiratze erabiliko du.
-    fa = قبل از شروع استفاده از برنامه ، لطفاً نمای کلی جهانی را در دستگاه خود بارگیری کنید.\nاز ٪@ ذخیره سازی استفاده می کند.
+    fa = قبل از استفاده از اپلیکیشن, اجازه دهید تا ما نقشه جهانی را بر روی موبایل شما دانلود کنیم.\nمقدار %@ از حافظه شما اشغال می شود.
```
The string should have a `@%` placeholder. Your new version had it changed to `٪@` which caused an error. Same with zh-Hant.

hemanggs replied:
yes that makes sense , thank you

pastk commented on iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings:
```diff
@@ -1792,7 +1792,7 @@
 "type.barrier.chain" = "Cadena";
 
-"type.barrier.city_wall" = "Muralla";
+"type.barrier.city_wall" = "City Wall";
```
This change shouldn't be here..

pastk commented on data/strings/strings.txt:
```diff
@@ -1571,49 +1571,48 @@
-    es = Antes de comenzar, permita que descarguemos en su dispositivo un mapamundi general.\nSe requieren %@ de almacenamiento.
-    et = Ennem rakenduse kasutamise alustamist luba meil laadida seadmesse alla maailma üldkaart.\nSee kasutab %@ mälumahtu.
-    eu = Hasi aurretik deskargatu mesedez mundu-mapa orokorra zure gailura.\nMemorian %@ erabiliko ditu.
+    en = Before you start using the app, please download the general world overview to your device.\nIt will use %@ of storage.
```
IMO dropping the "map" is not a good idea. What is a "world overview"? How would a new user make sense of it?

```suggestion
    en = Before you start using the app, please download the world overview map to your device.\nIt will use %@ of storage.
```

pastk replied:
Applies to all languages I think.

hemanggs replied:
Agreed

pastk commented on data/strings/strings.txt:
```diff
@@ -29965,41 +29964,49 @@
-    uk = Завантажити карту світу
-    zh-Hans = 下载世界地图
-    zh-Hant = 下載世界地圖
+    en = Download the world overview
```
```suggestion
    en = Download the world overview map
```

pastk reviewed (changes requested):
Actually I'd suggest to change only `en` strings to set a new guideline. And let translators update their languages as needed.
(the new automatic translations are not very good)

map-per commented on data/strings/strings.txt:
```diff
@@ -1571,49 +1571,48 @@
+    ca = Abans de començar a utilitzar l’aplicació, descarregueu la visió general del món general al vostre dispositiu.\nUtilitzarà %@ d’emmagatzematge.
+    cs = Než začnete aplikaci používat, stáhněte si do svého zařízení obecný přehled světa.\nVyužije %@ úložného prostoru.
+    da = Før du begynder at bruge appen, skal du downloade den generelle verdensoversigt til din enhed.\nDet vil bruge %@ af lagerpladsen.
+    de = Bevor Sie die App verwenden, laden Sie bitte die allgemeine Weltübersicht auf Ihr Gerät herunter.\nEs wird %@ des Speicherplatzes verwenden.
```
```suggestion
    de = Bevor Sie die App verwenden, laden Sie bitte die weltweite Übersichtskarte herunter.\nEs werden %@ Speicherplatz benötigt.
```

(already includes "map" and excludes "general" as suggested in https://github.com/organicmaps/organicmaps/pull/10177#discussion_r1938383291)

map-per commented on data/strings/strings.txt:
```diff
@@ -29965,41 +29964,49 @@
+    ca = Descarregueu la visió general del món
+    cs = Stáhněte si přehled světa
+    da = Download verdensoversigten
+    de = Laden Sie die Weltübersicht herunter
```
```suggestion
    de = Lade die weltweite Übersichtskarte herunter
```

map-per reviewed (commented):
DE translations

pastk commented on data/strings/strings.txt:
```diff
@@ -1571,50 +1571,49 @@
+    pt = Antes de começar a utilizar a aplicação, descarregue a visão geral do mundo para o seu dispositivo.\nUtilizará %@ do armazenamento.
+    pt-BR = Antes de você começar a usar o aplicativo, faça o download da visão geral do mundo para o seu dispositivo.\nEle usará %@ do armazenamento.
+    ro = Înainte de a începe să utilizați aplicația, vă rugăm să descărcați prezentarea generală a lumii pe dispozitivul dvs.\nAcesta va utiliza %@ din spațiul de stocare.
+    ru = Прежде чем начать пользоваться приложением, загрузите на свое устройство общий обзор мира.\nОн будет использовать %@ хранилища.
```
`ru = Для начала работы скачайте, пожалуйста, обзорную карту мира на ваше устройство.\nПотребуется %@ данных.`

pastk commented on data/strings/strings.txt:
```diff
@@ -29965,41 +29964,49 @@
+    pt = Descarregue a panorâmica mundial
+    pt-BR = Faça o download da visão geral do mundo
+    ro = Descărcați prezentarea generală a lumii
+    ru = Скачать обзор мира
```
`ru = Скачать обзорную карту мира`

hemanggs commented:
- Removed past commits with rebase for cleaner commit history  ( the PR closed automatically for this reason i assume , oops! ) 


**Suggested Changes from comments Applied :** 
- Reverted all translations to original while only changing `en` as suggested in [comment](https://github.com/organicmaps/organicmaps/pull/10177#pullrequestreview-2588445446) and [comment](https://github.com/organicmaps/organicmaps/pull/10177#discussion_r1938383291)
- Applied Translation suggestions from this PR's comments : 
 [fr](https://github.com/organicmaps/organicmaps/pull/10177#discussion_r1938351797), [PT](https://github.com/organicmaps/organicmaps/pull/10177#discussion_r1938375714), [DE](https://github.com/organicmaps/organicmaps/pull/10177#pullrequestreview-2588489786), [ru](https://github.com/organicmaps/organicmaps/pull/10177#discussion_r1938432769)
- Avoided unwanted [auto generated localization](https://github.com/organicmaps/organicmaps/pull/10177#discussion_r1938383118)

pastk commented:
@hemanggs are you happy with the PR merged as is?

Or if you're keen then you can mention `@organicmaps/translations` team so that other langs could be reviewed/updated. Then you'll need to integrate suggested changes.

hemanggs commented:
> @hemanggs are you happy with the PR merged as is?
> 
> Or if you're keen then you can mention `@organicmaps/translations` team so that other langs could be reviewed/updated. Then you'll need to integrate suggested changes.

Glad to add further suggestions , theres no hurry .

i think i have to join the translations team to be able to tag , i have posted comment [here](https://github.com/orgs/organicmaps/discussions/8538#discussioncomment-12038819)

csidirop commented on android/app/src/main/res/values-de/strings.xml:
You used informal and formal speech here. We should be consistent here, no matter which variant you choose. At least until we have decided whether we use the formal or informal speech app wide.

csidirop commented on android/app/src/main/res/values-de/strings.xml:
```diff
@@ -779,7 +779,7 @@
     <string name="isolines_toast_zooms_1_10">Karte vergrößern, um Höhelinien sichtbar zu machen</string>
     <string name="downloader_loading_ios">Herunterladen</string>
-    <string name="download_map_title">Herunterladen der Weltkarte</string>
+    <string name="download_map_title">Lade die weltweite Übersichtskarte herunter</string>
```
For the moment I would change it to the formal speech, because we used it in the other sentences. 
```suggestion
    <string name="download_map_title">Laden Sie die weltweite Übersichtskarte herunter</string>
```

map-per replied:
:+1: I used informal language because #8465 is likely to be merged. But one formal and one informal was not very consistent.

csidirop reviewed (changes requested):
I like the rephrasing for german but one line is uses formal and one line uses informal speech.

matheusgomesms commented on data/strings/strings.txt:
```diff
@@ -29993,7 +29993,7 @@
     pt = Descarregar o mapa mundial
     pt-BR = Baixar mapa mundial
```
```suggestion
    pt = Descarregar a visão geral mundial
    pt-BR = Baixar a visão geral mundial
```

matheusgomesms replied:
Forgot to review these 2 lines :)

andergd commented:
> @organicmaps/translations please review and update. The change is basically from "world map" to "world overview map".

Greetings. Can I have some context of this "world overview map"? Thanks

hemanggs commented:
@andergd Fixes https://github.com/organicmaps/organicmaps/issues/9334

andergd commented on data/strings/strings.txt:
```diff
@@ -29965,21 +29965,21 @@
-    de = Herunterladen der Weltkarte
+    de = Laden Sie die weltweite Übersichtskarte herunter
     el = Κατέβασμα παγκόσμιου χάρτη
     es = Descarga el mapa mundial
```
Descarga el mapa mundial de previsualización

andergd reviewed (commented):
Some other translator could improve these two, especially the Spanish one

soshial commented on data/strings/strings.txt:
```diff
@@ -1571,7 +1571,7 @@
   [download_resources]
     tags = android
-    en = Before you start using the app, please download the general world map to your device.\nIt will use %@ of storage.
+    en = Before you start using the app, please download the world overview map to your device.\nIt will use %@ of storage.
```
lv = Pirms sākt izmantot lietotni lejupielādējiet pasaules pārskata karti.\nTā ierīces krātuvē aizņems %@ vietas.

soshial commented on data/strings/strings.txt:
```diff
@@ -29965,21 +29965,21 @@
   [download_map_title]
     tags = android
-    en = Download the world map
+    en = Download the world overview map
```
lv = Lejupielādēt pasaules pārskata karti

Jean-BaptisteC reviewed (approved):
LGTM for french

soshial commented on data/strings/strings.txt:
```diff
@@ -1596,16 +1596,16 @@
     ko = 시작하시기 전에 일반 세계 지도를 귀하의 장치로 다운로드하겠습니다.\n%@ 데이터가 필요합니다.
     lt = Leiskite mums atsisiųsti bendrąjį pasaulio žemėlapį į jūsų įrenginį prieš pradėdami naudotis programėle.\nTai užims %@ saugyklos.
-    lv = Pirms lietotnes izmantošanas lejupielādējiet kopējo pasaules karti.\nTā ierīce krātuvē aizņems %@ vietas.
+    lv = lv = Pirms sākt izmantot lietotni lejupielādējiet pasaules pārskata karti.\nTā ierīces krātuvē aizņems %@ vietas.
```
This shouldn't have `lv = lv =`. Guys, when will we migrate to Weblate so that these mistakes don't occur? @pastk @biodranik @rtsisyk

[Carthago delenda est](https://en.wikipedia.org/wiki/Carthago_delenda_est)

pastk replied:
Well, as soon as someone will do all the migration work..

pastk commented:
@hemanggs many thanks!
